contrib/buildrpm/buildrpmLibfabric.sh

```diff
@@ -224,16 +236,33 @@ if [[ -z "$tardirname" ]]; then
 fi
 verbose "Name of package: $tardirname"
 
-version=$(basename "$tardirname" | cut -d- -f2)
-if [[ -z "$tardirname" ]]; then
-  error 4 "Cannot determine version from $tarball"
+if [[ -z "$version" ]] ; then
+  version=$(basename "$tardirname" | cut -d- -f2)
+  if [[ -z "$tardirname" ]]; then
```

Comment:
What is the purpose of checking tardirname here?
This is already being done at line 234.
Do you want to check 'version' again?

Reply:
As I understand the usage, the prior access of 'tardirname' was used to retrieve the tarball name. The access of tardirname here is used to retrieve the libfabric version. Beyond that, I'd ask @jsquyres. He might know.

Reply:
Yes, you are getting the version from the tardirname.  But why are you checking tardirname for a zero length after trying to get the version number from tardirname?  You should be checking version.

Reply:
I see. Yes, that makes sense.

Reply:
Agreed -- you should be checking `$version` for error here.

Reply:
@jsquyres , @tonyzinger : I've updated the PR with the fix, plus documentation to the -h/help option for the new options.